backend/routes/posts.route.js

```diff
@@ -5,9 +5,11 @@ const authUtils = require('../utils/auth.js')
 
 router.post('/', authUtils.isAuthenticated, PostsController.createNewPost)
 
-//GET ROUTE for post detail by post id - READ OPERATION
-router.get('/getById/:id', PostsController.getPostDetails)
+
+router.get('/getById/:id', PostsController.getPostDetails) //get post detail by post id
 router.get('/getByUser', authUtils.isAuthenticated, PostsController.getUsersPosts) // get posts of authenticated user
 router.get('/getByUser/:id', authUtils.isAuthenticated, PostsController.getOtherUsersPosts) // get posts of another user
+router.get('/deleteById/:id', authUtils.isAuthenticated, PostsController.deletePost) // get to delete post by id
+router.post('/editById/:id',authUtils.isAuthenticated, PostsController.editPost)// post route to edit post by id
```

Comment:
please make this a put request instead of post since it updates, and delete instead of get